src/app/question/components/RadioGroup.tsx

```diff
@@ -0,0 +1,41 @@
+export default function RadioGroup({ active, onChange }: {
+  active: number; onChange: (num: number) => void;
+}) {
+  const textLabel = ['매우 그렇지 않다', '그렇지 않다', '보통이다', '그렇다', '매우 그렇다'];
+
+  return (
+    <div>
+      {[1, 2, 3, 4, 5].map((num) => (
+        <label
+          key={num}
+          htmlFor={`option-${num}`}
+          className={`
+            w-full block mb-2.5 cursor-pointer
+          `}
+        >
+          <input
+            type="radio"
+            id={`option-${num}`}
+            name="option"
+            value={num}
+            checked={active === num}
+            onChange={() => onChange(num)}
+            className="w-full peer hidden"
+          />
+          <div
+            className={`
+              w-full h-[3.13rem] flex items-center px-5 rounded-xl font-bold text-sm md:text-lg md:h-[3.75rem] md:px-7
+              transition-colors truncate
+              ${active === num
+                ? 'bg-[#14B8A6] text-white'
+                : 'bg-[#F3F3F7] text-black'
+              }
+            `}
+          >
```

Comment:
`className` 내에 조건식을 넣으면
1. 무슨 의미의 조건인지 판단해야 하고
2. 전반적인 가독성이 떨어진다고 느껴서

저는 개인적으로 이 방식을 선호하지는 않는 편인데요.

여기도 tailwind의 유틸리티 클래스를 활용해서 조건식을 제거해볼 수 있을 거 같아요.
아래 내용 한 번 읽어보시고, 괜찮다면 적용해 주세요

---

`active === num` 조건은 `div`의 형제인 `input` 요소의 `checked` 상태를 알기 위한 조건으로 보여요.
[`peer-*`](https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-the-descendants-of-a-peer)를 활용하면 아래처럼 작성해볼 수 있겠네용

```suggestion
          <div
            className={`flex h-[3.13rem] w-full items-center truncate rounded-xl bg-[#F3F3F7] px-5 text-sm font-bold text-black transition-colors peer-checked:bg-[#14B8A6] peer-checked:text-white md:h-[3.75rem] md:px-7 md:text-lg`}
          >
```

형제 input이 checked 상태일 때만 스타일이 적용된다는 의도가 명확히 드러나서
`active === num`이 어떤 의미인지 따로 이해하지 않아도 된다는 점에서 더 직관적이라고 생각합니다!

(`div`에 `peer-*` 클래스를 사용하려면 `input`의 className에 `peer`를 넣어줘야 하는데,
이 부분은 이미 되어있네요! 원래는 어떤 의도로 작성하셨던 건가요? 👀 )

Reply:
그리고 div에 hex code 그대로 들어가 있는 부분도 수정 예정이라고 하셨던 거 같은데 맞나용?